.github/release-drafter-lts12.yml

```diff
@@ -0,0 +1,42 @@
+name-template: 'v$RESOLVED_VERSION 👨‍🔧'
+tag-template: 'v$RESOLVED_VERSION'
+filter-by-commitish: true
+commitish: release/12.0
```

Comment:
same as master; despite this commitish filter

see https://github.com/release-drafter/release-drafter/issues/656#issuecomment-806184887